pkg/db/tx.go

```diff
@@ -52,19 +50,26 @@ func (d *dbTx) Query(query string, args ...interface{}) (pgx.Rows, error) {
 			"query": queryToString(query),
 		})
 	if err != nil {
-		log.WithError(err).Error("SQL query failed with error")
-		return nil, err
+		return nil, d.handleSQLError(err, "query", query)
 	}
-	log.Trace("SQL query started successfully")
+	log.Trace("SQL query executed successfully")
```

Comment:
```suggestion
	log.Trace("SQL query started successfully")
```

**Explanation:** The query has not yet been executed!  This function returns a `pgx.Rows`.  The caller will iterate over that to read each row.  So an error _can_ be returned, for instance if `query` is really broken... but the function returns before the query is over.